glossary.md

```diff
@@ -116,6 +116,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | fit                 | khớp           |               |
 | first principle     | định đề cơ bản |               |
 | functional anaylsis | giải tích hàm  |               |
+| fully-connected     | fully-connected  |               |
```

Comment:
```suggestion
| fully-connected     | fully-connected  |               |
```

Reply:
hay là... hoàn toàn?

Reply:
Nếu dịch thì t vẫn thích `đủ` hoặc `đầy đủ` hơn, chỉ phân vân có nên dịch không.

Reply:
Mình thấy nên dịch, thêm ý kiến nhỉ. 
cc @minhduc0711 @rootonchair @tiepvupsu

Reply:
Nếu mà dịch thì mình vote từ `đầy đủ` hơn...

Reply:
Em vote "kết nối đầy đủ" nhé

Reply:
mình cũng thấy `kết nối đầy đủ` ổn

Reply:
Mọi người đi qua cho cái emoji vote nhé:
- Kết nối đầy đủ 👍 
- Kết nối hoàn toàn ❤️ 
- Không dịch 🎉